gtfs/spec/en/reference.md

```diff
@@ -406,6 +408,21 @@ There must not be overlapping time intervals for the same `timeframe_group_id` a
 - The “current day” is the current date of the fare event’s time, computed relative to the local timezone.  The “current day” may be different from the service day of a fare leg’s trip, especially for trips that extend past midnight.
 - The “time-of-day” for the fare event is computed relative to “current day” using GTFS Time field-type semantics.
 
+### rider_categories.txt
+
+File: **Optional** 
+
+Primary key (`rider_category_id`)
+
+Defines categories of riders (e.g. elderly, student).
+
+|  Field Name | Type | Presence | Description |
+|  ------ | ------ | ------ | ------ |
+|  `rider_category_id` | Unique ID | **Required** | Identifies a rider category. |
+|  `rider_category_name` | Text | **Required** | Rider category name as displayed to the rider. |
+|  `is_default_fare_category` | Enum | **Required** | Specifies if an entry in [rider_categories.txt](#rider_categoriestxt) should be considered the default category (i.e. the main category that should be displayed to riders). For example: Adult fare, Regular fare, etc. Valid options are:<br><br>`0` or empty - Category is not considered the default.<br>`1` - Category is considered the default one.<br><br>When multiple rider categories are eligible for a single fare product specified by a `fare_product_id`, there must be only one of these rider categories indicated as the default rider category (`is_default_fare_category = 1`). |
```

Comment:
This could be a bit clearer (as well as in the definition for `fare_products.rider_category_id`), should it be "there must be at most one" or "there must be exactly one"? In other words, should it be permissible to not have any categories as `is_default_fare_category = 1`?

Reply:
I think the idea is that multiple entries in `rider_categories.txt` may have `is_default_fare_category = 1`. However, when a `fare_product_id` in `fare_products.txt` references multiple `rider_category_id`s, exactly one of these referenced `rider_category_id`s must have `is_default_fare_category = 1`. When the `rider_categories.txt` file exists, it is not allowed to have no categories set as `is_default_fare_category = 1`.

Does it look clearer if we just modify the sentence from
> there must be only one of these rider categories indicated as the default rider category (`is_default_fare_category = 1`).

to

there must be **exactly** one of these **referenced** rider categories indicated as the default rider category (`is_default_fare_category = 1`)?

Reply:
Thanks, @tzujenchanmbd, the explanation makes a lot of sense, I think in my initial reading I missed the context that this was among those rider categories filtered for a specified fare product ID.

I like your proposed clarification, though might suggest changing "referenced" to "eligible" to make even more clear which rider categories we're talking about.